layouts/partials/head.html

```diff
@@ -2,9 +2,23 @@
   <meta charset="utf-8">
   <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
   <meta name="viewport" content="width=device-width">
+
   <meta name="project" content="{{ .Params.project }}">
   <meta name="version" content="{{ .Params.project_version }}">
-  <meta name="base-url" content="/{{ .Scratch.Get "site_relative_url" }}">
+
+  <!-- TODO: Adding these leading slashes is... pretty dumb. IMO. We should
```

Comment:
So in this case, `site_relative_url` is going to be something like `riak/kv/2.1.4/installing/` even thought that's not actually a site-relative url. `/riak/kv/2.1.4/installing/` (note the leading `/`) is a site-relative url, and we want to make sure we record the correct thing in the meta tag.